discovery-provider/src/app.py

```diff
@@ -363,6 +363,7 @@ def configure_celery(flask_app, celery, test_config=None):
             "src.tasks.cache_trending_playlists",
             "src.tasks.index_solana_plays",
             "src.tasks.index_aggregate_views",
+            "src.tasks.index_aggregate_user",
```

Comment:
should we just make this part of the existing aggregate task as opposed to this new one?

Reply:
oh right, gotcha. this task already exists